supplier/models.py

```diff
@@ -0,0 +1,121 @@
+from django.db import models
+from django.core.validators import MinLengthValidator
+from django.core.exceptions import FieldDoesNotExist, ObjectDoesNotExist, ValidationError
+
+
+class User(models.Model):
```

Comment:
I don't think the User is part of the supplier app. will we create the same user for the Client app?

Reply:
yes, its an abstract class, is there a good reason to create it in separate module/app?

Reply:
The User's in this case are the same table for both Supplier and Client. if you plan to have 1 app for it, it has to be a users app that'll include all three, or separate them, AFAIK.

Reply:
like above, 2 separated apps, both include User table